docs/annexes/SPDX-license-expressions.md

```diff
@@ -1,6 +1,6 @@
-# Annex D: SPDX license expressions (Normative)
+# SPDX license expressions (Normative)
 
-## D.1 Overview <a name="D.1"></a>
+## Overview <a name="D.1"></a>
```

Comment:
```suggestion
## Overview
```
If subsection reference is not desirable, removing the HTML anchor.

Reply:
For cleaner URLs, propose to rename these files:
- `SPDX-Lite.md` to `spdx-lite.md`
- `SPDX-license-expressions.md` to `spdx-license-expressions.md`

Other files are already in all-lowercase, including ones with a word which originally an all-uppercase initialism:
- `changes-from-previous-iso.md` (ISO)
- `pkg-url-specification.md` (URL)
- `rdf-model.md` (RDF)

The all-lowercase `spdx` will also match the other `spdx` instances in the full URL of: `https://spdx.github.io/spdx-spec/...`

Reply:
I will put this in another PR.